lib/network.sh

```diff
@@ -118,6 +110,23 @@ else
     exit 1
 fi
 
+if [[ -n "${CLUSTER_APIENDPOINT_IP}" ]]; then
+  # Accept user provided value if set
+  export CLUSTER_APIENDPOINT_IP
+elif [[ -n "${EXTERNAL_SUBNET_V4}" ]]; then
+  network_address CLUSTER_APIENDPOINT_IP "${EXTERNAL_SUBNET_V4}" 249
+else
+  network_address CLUSTER_APIENDPOINT_IP "${EXTERNAL_SUBNET_V6}" 249
+fi
+
+# shellcheck disable=SC2153
+if [[ "$CLUSTER_APIENDPOINT_IP" == *":"* ]]; then
```

Comment:
This one is missing curlys around the variable, vs the others above it. Lines 124 and 126 have the same as well.